ferretdb_packaging/defineversion/main.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main defines debian version number of DocumentDB for CI builds.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"slices"
+	"strings"
+
+	"github.com/sethvargo/go-githubactions"
+)
+
+func main() {
+	controlFileF := flag.String("control-file", "../pg_documentdb/documentdb.control", "pg_documentdb/documentdb.control file path")
+
+	flag.Parse()
+
+	action := githubactions.New()
+
+	if *controlFileF == "" {
+		action.Fatalf("%s", "-control-file flag is empty.")
+	}
+
+	controlDefaultVersion, err := getControlDefaultVersion(*controlFileF)
+	if err != nil {
+		action.Fatalf("%s", err)
+	}
+
+	debugEnv(action)
+
+	upstreamVersion, err := define(controlDefaultVersion, action.Getenv)
+	if err != nil {
+		action.Fatalf("%s", err)
+	}
+
+	setResults(action, upstreamVersion)
+}
+
+// controlDefaultVer matches major, minor and patch from default_version field in control file,
```

Comment:
```suggestion
// controlDefaultVer matches major, minor and "patch" from default_version field in control file,
```

As it is not really a patch

Reply:
Thanks for explanation :pray: